lib/stdlib/src/ordsets.erl

```diff
@@ -67,6 +67,20 @@ size(S) -> length(S).
 
 is_empty(S) -> S=:=[].
 
+%% is_equal(OrdSet1, OrdSet2) -> boolean().
+%%  Return 'true' if OrdSet1 and OrdSet2 contain the same elements,
+%%  otherwise 'false'.
+-spec is_equal(Ordset1, Ordset2) -> boolean() when
+      Ordset1 :: ordset(_),
+      Ordset2 :: ordset(_).
+
+is_equal([E1|S1], [E2|S2]) ->
+    E1 == E2 andalso is_equal(S1, S2);
+is_equal([], []) ->
+    true;
+is_equal(S1, S2) when is_list(S1), is_list(S2) ->
+    false.
```

Comment:
`S1 == S2`... it never occured to me that this would work this way :sweat_smile: Makes sense though, and it is indeed documented. Knowing this will also make a few things easier in #7256 :smiley: